constellations/gemini-admin/src/classes/classEdit.js

```diff
@@ -273,8 +303,14 @@ export class ClassEditPage extends React.Component {
                             this.handleChange(e, "inputTimeString")
                         }
                     />
-                </div>
+                    <h3 className="avalibility">Class Avalibility</h3>
 
+                    <select onChange={(e) => this.onChangeSelect(e)}>
+                        <option value="0">Normal</option>
+                        <option value="1">Almost Full</option>
+                        <option value="2">Full</option>
+                    </select>
```

Comment:
I feel like it would make more sense to have this in the class information section rather than the schedule section